### DISCUSSION TITLE
Creating an assistant

### DISCUSSION THREAD

**Discussion by cdahllof** [MacPaw/OpenAI#212] [Q&A] [answered]
Hello,

I am trying to create an assistant. Is that possible?

OpenAI info about assistant.

https://platform.openai.com/docs/assistants/how-it-works/creating-assistants?lang=curl

Thanks!

**Comment by benadamdev** [comment, marked as answer]
Hi,
This library doesn't support assistants yet. The only Open API Swift library that does so far is https://github.com/jamesrochabrun/SwiftOpenAI.

**Reply by cdahllof** [reply]
Thank you.
